run/SimExamples/Signal_ImpactB/README.md

```diff
@@ -0,0 +1,12 @@
+This is a simulation example showing the following things
```

Comment:
We need to put some doxygen hooks in README like this. Please take a look in the other examples.